src/notification/index.js

```diff
@@ -79,10 +79,8 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
-      unique_identities_count: new Set(urlMatches.map(matchIndex => identities[matchIndex].email))
-        .size,
+      unique_accounts_count: new Set(urlMatches.map(matchIndex => accounts[matchIndex].email)).size,
     });
-    // TODO get user_id into accounts data, and use that
```

Comment:
nit: this TODO comment should go away in the later commit where we start using the user_id